source/api/commands/writefile.md

```diff
@@ -54,6 +54,7 @@ Pass in an options object to change the default behavior of `cy.writeFile()`.
 Option | Default | Description
 --- | --- | ---
 `log` | `true` | {% usage_options log %}
+`flag` | `w` | File system flag as used with [`fs.writeFile`](https://nodejs.org/api/fs.html#fs_file_system_flags)
```

Comment:
> 👍 for the linkage.

The `{% url %}` tag helper that Jennifer and others have created will validate that the URL exists so I believe its use is preferred to regular Markdown linking. See `TAGS.md` at the root of this repo for the lowdown on the `url` tag (and others).

```md
File system flag as used with {% url `fs.writeFile` https://nodejs.org/api/fs.html#fs_file_system_flags %}
```

> This also applies to the hyperlink further down on line `#142`.